src/lib/ipfs-utils.js

```diff
@@ -0,0 +1,16 @@
+import { IPFS_ENDPOINT } from '../endpoints'
+
+const REQUEST_TIMEOUT = 60000
+
+export const ipfsGet = async hash => {
```

Comment:
```suggestion
export const ipfsGet = async cid => {
```

Is a bit more semantically correct :).